docs/_data/sidebar.yml

```diff
@@ -3,9 +3,9 @@ main:
     title: Welcome
     path: /
     items:
-      why-conjur:
-        title: Why Conjur
-        path: /welcome/why-conjur.html
+      # why-conjur:
+      #   title: Why Conjur
+      #   path: /welcome/why-conjur.html
```

Comment:
Not sure if it matters but you can still get to this page manually by visiting the URL directly. We could just delete this (and the associated page) instead of commenting if we want to be absolutely sure it's gone. The nice thing about version control is you can always look back! 😄

Reply:
good catch! I'll remove that now